Sources/Client/Util/Updater.swift

```diff
@@ -160,12 +171,12 @@ public final class Updater: ObservableObject {
   // MARK: Helper
   
   /// - Returns: A download URL and a version string
-  public static func getDownloadURL(_ type: UpdateType) throws -> (URL, String) {
+  public static func getDownloadURL(_ type: UpdateType, unstableBranch: String) throws -> (URL, String) {
```

Comment:
Given that this function is only called once, it might be best just to inline it (manually). I say this because having the `unstableBranch` argument even when getting the download URL for stable is a bit odd.